LykkeApi2/Controllers/ClientController.cs

```diff
@@ -217,11 +217,18 @@ public class ClientController : Controller
                 return StatusCode((int) HttpStatusCode.InternalServerError);
             }
 
+            var country = (await _personalDataService.GetAsync(_requestContext.ClientId)).Country;
+
+            if (string.IsNullOrEmpty(country))
```

Comment:
Please move this to Personal Data Service?

Reply:
We don't need this for LWW-56 - to be removed